colobot-base/src/ui/controls/edit.cpp

```diff
@@ -1617,7 +1616,7 @@ bool CEdit::ReadText(const std::filesystem::path& filename)
             if ( m_bSoluce || !bInSoluce )
             {
                 HyperLink link;
-                link.name = GetNameParam(buffer.data()+i+3, 0);
+                link.name = StrUtils::ToPath(GetNameParam(buffer.data()+i+3, 0));
```

Comment:
Many SatCom links are broken here most likely due to help files using backslash in file paths.

OS: Xubuntu 24.04.1 LTS

Reply:
This line from log seems to show backslash as part of the filename:

> [ERROR]: Failed to load text file help/E/cbot\radarall.txt

Reply:
Found it. I removed this line:

https://github.com/colobot/colobot/blob/84ac0d3ba8bc7c590da0b999f132f765f6d56ed1/src/ui/controls/edit.cpp#L819

Will fix soon